src/electionguard/key_ceremony.py

```diff
@@ -179,49 +201,80 @@ def generate_election_key_pair(
 
 
 def generate_election_partial_key_backup(
-    owner_id: GuardianId,
-    polynomial: ElectionPolynomial,
-    designated_guardian_key: ElectionPublicKey,
+    sender_guardian_id: GuardianId,
+    sender_guardian_polynomial: ElectionPolynomial,
+    receiver_guardian_public_key: ElectionPublicKey,
 ) -> ElectionPartialKeyBackup:
     """
     Generate election partial key backup for sharing
-    :param owner_id: Owner of election key
-    :param polynomial: The owner's Election polynomial
+    :param sender_guardian_id: Owner of election key
+    :param sender_guardian_polynomial: The owner's Election polynomial
+    :param receiver_guardian_public_key: The receiving guardian's public key
     :return: Election partial key backup
     """
-    value = compute_polynomial_coordinate(
-        designated_guardian_key.sequence_order, polynomial
+    coordinate = compute_polynomial_coordinate(
+        receiver_guardian_public_key.sequence_order, sender_guardian_polynomial
+    )
+    coordinate_data = CoordinateData(coordinate)
+    nonce = rand_q()
+    seed = get_backup_seed(
+        receiver_guardian_public_key.owner_id,
+        receiver_guardian_public_key.sequence_order,
+    )
+    encrypted_coordinate = hashed_elgamal_encrypt(
+        coordinate_data.to_bytes(),
+        nonce,
+        receiver_guardian_public_key.key,
+        seed,
     )
     return ElectionPartialKeyBackup(
-        owner_id,
-        designated_guardian_key.owner_id,
-        designated_guardian_key.sequence_order,
-        value,
+        sender_guardian_id,
+        receiver_guardian_public_key.owner_id,
+        receiver_guardian_public_key.sequence_order,
+        encrypted_coordinate,
     )
 
 
+def get_backup_seed(receiver_guardian_id: str, sequence_order: int) -> ElementModQ:
+    return hash_elems(receiver_guardian_id, sequence_order)
```

Comment:
I wasn't sure, is the guardian's id and sequence order sufficient for the backup seed?

Reply:
This is up to the specification. I don't think this will likely be enough, but this is a question for @benaloh